sdk/analytics/src/test/java/com/klaviyo/analytics/networking/requests/BaseApiRequestTest.kt

```diff
@@ -16,7 +16,7 @@ internal abstract class BaseApiRequestTest<T> : BaseTest() where T : KlaviyoApiR
     open val expectedHeaders = mapOf(
         "Content-Type" to "application/json",
         "Accept" to "application/json",
-        "Revision" to "2023-07-15",
+        "Revision" to "1234-56-78",
```

Comment:
bring me back to the 1200s

Reply:
Don't worry, our [API revision policies](https://developers.klaviyo.com/en/docs/api_versioning_and_deprecation_policy#fall-forward-behavior) will fall forward 790 years